### PR TITLE
Parse end device QR code via QR code generator parse endpoint

### LINKED ISSUE #6059: Console manually parses the QR code
### Summary

The QR code information parser in the Console manually parses the QR codes based on TR005 knowledge, but it does not have to do it and for draft versions the parser is wrong.

### Steps to Reproduce

1. Attempt to scan a TR005 draft QR code. An example is `URN:DEV:LW:70B3D57ED0000000_0004A31000FFFFFF_00000000_VD53FFFFF`.
2. Try to claim the device.

### Current Result

The claim authentication code (owner token) parsed used by the Console is `VD53FFFFF`.

### Expected Result

The authentication code in the drafts actually is `D53FFFFF` (the `V` must be dropped).

But the general remark here is that instead of [the manual parser](https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.24/pkg/webui/console/lib/device-claiming-parse-qr.js) the backend parser should be used.

The backend parser is exposed by the `EndDeviceQRCodeGenerator` service:
https://github.com/TheThingsNetwork/lorawan-stack/blob/185a4f82d10ef1ade8a463f2b24351a7eb93b6a6/api/qrcodegenerator.proto#L108-L118
It can parse the QR code contents and ensure that we don't need to duplicate the QR code format logic between the frontend and the backend.

### Relevant Logs

_No response_

### URL

_No response_

### Deployment

The Things Stack Enterprise (self-hosted)

### The Things Stack Version

3.24.1

### Client Name and Version

_No response_

### Other Information

_No response_

### Proposed Fix

Use the API instead of manually parsing the contents of the QR code.

### Contributing

- [X] I can help by doing more research.
- [ ] I can help by implementing a fix after the proposal above is approved.
- [X] I can help by testing the fix before it's released.

### Code of Conduct

- [X] I agree to follow TTN's [Community Code of Conduct](https://github.com/TheThingsNetwork/Manifesto/blob/master/CodeOfConduct.md).

### PR DESCRIPTION
**Pull request by mjamescompton**
#### Summary

Closes #6059 

#### Changes

- Add QR code generator to jsSDK
- Use parse endpoint instead of js function when scanning the QR code.
- Remove old js parse functions

#### Testing

Setting as draft to see the tests run, there is no change to the user so the tests should pass. See below tested locally using random QR code and valid QR code.

##### Regressions

no

#### Notes for Reviewers

Random text

62E4AB14-E00C-4192-B96D-A476170E3E7F_1_105_c

Valid QR code

F94D4AC0-1C61-4D6A-A916-ACEF011E85B9_1_105_c

This passes regardless of weather the device the QR code is related to exists. This validation happen when the end device registration form is submitted.

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [ ] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Comment by mjamescompton** [comment]
@kschiffer The fail on the end to test is because the backend does not return the `lora_alliance_profile_ids` we use on on the end to end tests. The only solutions I see is we remove this test or we mock the response from this new endpoint. What would you prefer?

**Comment by kschiffer** [comment]
Please mock the response.

File: pkg/webui/console/store/middleware/logics/qr-code-generator.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// Copyright © 2022 The Things Network Foundation, The Things Industries B.V.
```
Comment (kschiffer) [inline pkg/webui/console/store/middleware/logics/qr-code-generator.js]:
```suggestion
// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
```

**Comment by mjamescompton** [comment]
@kschiffer all green, so please review

**Review by kschiffer — APPROVED** [review]
LGTM

**Pull request merged**